examples/nextjs/package.json

```diff
@@ -9,10 +9,14 @@
     "lint": "next lint"
```

Comment:
This template is auto generated via cron job so we will need to update that to use `pnpm` as well or it will just be overridden on the next run. 

x-ref: https://github.com/vercel/vercel/blob/d8d30b59ec622780f3e67118bb373efd12ce0f00/utils/update-next.js#L35

Reply:
Okay, will reset that change here for now.